src/blueprint/azext_blueprint/custom.py

```diff
@@ -46,14 +46,16 @@ def import_blueprint_with_artifacts(cmd,
                 body['resource_groups'] = blueprint_properties['resourceGroups']  # dictionary
 
         for filename in os.listdir(os.path.join(input_path, 'artifacts')):
-            artifact_name = filename.split('.')[0]
-            filepath = os.path.join(input_path, 'artifacts', filename)
-            with open(filepath) as artifact_file:
-                try:
-                    artifact = json.load(artifact_file)
-                    art_dict[artifact_name] = artifact
-                except json.decoder.JSONDecodeError as ex:
-                    raise CLIError('JSON decode error for {}: {}'.format(filepath, str(ex))) from ex
+            # skip hidden files
+            if not filename.startswith('.'):

```

Comment:
Could we assume the files starting with `.` are all hidden files? Seems it's not guaranteed.
Also we need consider different plaforms.

Reply:
Adding the author @simonkeyd